build.sbt

```diff
@@ -2743,6 +2744,47 @@ lazy val runtime = (project in file("engine/runtime"))
   .dependsOn(`connected-lock-manager`)
   .dependsOn(testkit % Test)
 
+lazy val `runtime-and-langs` = (project in file("engine/runtime-and-langs"))
```

Comment:
It is better to provide `runtime-and-langs` as an aggregate project, similar to `enso`. It is afterall, merely an aggregation of multiple projects. You don't need any additional settings for that, you just need to specify the `aggregate`.